cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -576,6 +576,48 @@ def test_self_reported_ethnicity_ontology_term_id__unknown(self, validator_with_
         assert validator.validate_adata()
         assert validator.errors == []
 
+    def test_cell_type_ontology_term_id__unknown(self, validator_with_adata):
+        """
+        Test 'unknown' cell_type_ontology_term_id is valid
+        """
+        validator = validator_with_adata
+        obs = validator.adata.obs
+        obs.at["Y", "cell_type_ontology_term_id"] = "unknown"
+        assert validator.validate_adata()
+        assert validator.errors == []
+
+    def test_tissue_ontology_term_id__unknown(self, validator_with_adata):
+        """
+        Test 'unknown' tissue_ontology_term_id is valid if tissue_type is 'cell culture'
+        """
+        validator = validator_with_adata
+        obs = validator.adata.obs
+
+        # Arrange -- relies on "tissue_type" value for index "Y" being "cell culture", set explicitly
+        obs.at["Y", "tissue_type"] = "cell culture"
+        obs.at["Y", "tissue_ontology_term_id"] = "unknown"
+
+        assert validator.validate_adata()
+        assert validator.errors == []
+
+    def test_tissue_ontology_term_id__unknown_invalid(self, validator_with_adata):
+        """
+        Test 'unknown' tissue_ontology_term_id is valid if tissue_type is 'cell culture'
```

Comment:
i'm guessing this comment was copied and pasted, but can you update this to reflect what the test actually does? it would also be fine to remove this entirely, since the comment on line 610 explains the test sufficiently imo

Reply:
thank you 🙏 😅